libs/blocks/modal/modal.js

```diff
@@ -34,7 +34,9 @@ export function sendAnalytics(event) {
 function closeModal(modal) {
   const { id } = modal;
   const closeEvent = new Event('milo:modal:closed');
-  document.body.classList.remove('modal-open');
+  if (document.body.classList.contains('modal-open')) {
```

Comment:
Is this being used in an intersection observer context? Why exactly is the check needed? Otherwise `document.body.classList.remove('mumbo-jumbo')` will not result in an error, it will just not achieve anything.